### PR TITLE
Structured Logging + Dependency Pass

### LINKED ISSUE #1716: Support Structured Logging in the V4 Async Daemon
I think this is a great idea, and I'm trying to nominate @Hawxy for this one:)

Just gotta make sure that the `ILogger` usage is still okay for folks that aren't gonna sure Seq or the like.

### PR DESCRIPTION
**Pull request by Hawxy**
Closes #1716 

I also noticed some churn across the npgsql and JasperFx ecosystem so bumped everything to latest while I'm here. Seems to all be working fine.

### CODE REVIEW

File: src/Marten.CommandLine/Marten.CommandLine.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
     <PropertyGroup>
         <Description>Command line tooling for Marten</Description>
-        <TargetFramework>netstandard2.1</TargetFramework>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (mysticmind) [inline src/Marten.CommandLine/Marten.CommandLine.csproj:5]: Any specific reason why the target framework changed from `netstandard2.1` to `netcoreapp3.1`?

**Review by mysticmind — APPROVED** [review]
@Hawxy Looks good to me. Added another comment/question, please clarify.

File: src/Marten.CommandLine/Marten.CommandLine.csproj (hunk 1 above)
Comment (Hawxy) [reply src/Marten.CommandLine/Marten.CommandLine.csproj:5]: The targeting of `Oakton.AspNetCore` changed to `netcoreapp3.1` and `net5.0` only.

Comment (mysticmind) [reply src/Marten.CommandLine/Marten.CommandLine.csproj:5]: Thanks for the clarification @Hawxy 👍

**Comment by jeremydmiller** [comment]
@babuannamalai @Hawxy Looks good to me. I'll follow the same pattern for any additional work. Thanks for doing this and making the suggestion!

**Pull request merged**